src/commands/extract/print.ts

```diff
@@ -0,0 +1,57 @@
+import type { BaseExtractOptions } from '../extract';
+import { relative } from 'path';
+import { Command } from 'commander';
+
+import { extractKeysOfFiles } from '../../extractor';
+import { WarningMessages } from '../../utils/warnings';
+
+type ExtractPrintOptions = BaseExtractOptions;
+
+async function printHandler(this: Command, filesPattern: string) {
```

Comment:
Is there some reason why to call this function printHandler? and not just print?

Reply:
I just called all the command handlers "somethingHandler", for honestly no specific reason.